test/ClangImporter/availability_returns_twice.swift

```diff
@@ -0,0 +1,15 @@
+// RUN: %target-swift-frontend -typecheck -verify %s
+
+#if os(OSX) || os(iOS) || os(watchOS) || os(tvOS)
+  import Darwin
+  typealias JumpBuffer = Int32
+#else
+  import Glibc
+  typealias JumpBuffer = __jmp_buf_tag
+#endif
+
+func test_unavailable_returns_twice_function() {
+  var x: JumpBuffer
```

Comment:
Can support for these be improved somehow? Alternatively, is there a good discussion on why this is imported weirdly on macOS?